File: java_programs/packer_unpacker/Program434.java
import java.util.*;
import java.io.*;

class Program434{

    public static void main(String A[])
    {
        try{
            
            Scanner sobj = new Scanner(System.in);
            System.out.println("Enter the name of the file that you want to create.");
            String Fname = sobj.nextLine();
            File fobj = new File(Fname);
            if(fobj.exists()){
                System.out.println("File already exists.");
            }
            else{
                fobj.createNewFile();
                System.out.println("file successfully created.");
            }
        }
        catch(IOException iobj){

        }
        catch(Exception eobj){

        }
    }
}

File: python_programs/packer_unpacker/Program434.py
def main():
    try:

        file_name = input("Enter the name of the file that you want to create.")
        with open(file_name, "x") as fobj:
            fobj.close()
            print("File created successfully.")
    except FileExistsError:
        print("File already exists.")
    except Exception as e:
        print("Exception while creating a new file ", e)


if __name__ == "__main__":
    main()